src/components/GetReport.jsx

```diff
@@ -1,76 +1,145 @@
 import React from "react";
 import { Formik, Form, Field } from "formik";
-import { GetResponseErrors } from "../utilities/CitysourcedResponseHelpers";
 import FormContainer from './FormContainer';
+import { SetFieldValues } from "../utilities/FormHelpers";
 import { GetReportByID } from '../services/authService';
+import '../css/inside-responsive.min.css';
```

Comment:
these css refernces, should probaly go on the index.html file in the public folder.

Reply:
Oh yeah, I added them for testing. I'll remove it

Reply:
i do think it might be possible to include them in the index, so that we have them, but it doesn't impact deployment. Might be worth trying out at a future date.